src/Uno.UWP/Devices/Sensors/SimpleOrientationSensorOrientationChangedEventArgs.cs

```diff
@@ -3,7 +3,7 @@
 namespace Windows.Devices.Sensors
 {
 	/// <summary>
-	/// Provides data for the sensor reading�changed event.
+	/// Provides data for the sensor reading–changed event.
```

Comment:
Looks like an encoding issue?

Reply:
Yes. This was caught due to a [bug](https://developercommunity.visualstudio.com/t/New-version-of-Visual-Studio-for-Mac-17/10114754) 😄 

Without changing this file, the build would fail.

Reply:
Oh no, this wasn't showing like this when I edited the file. It was the `-` that's not showing correctly and I fixed it.

Reply:
Should be fixed now. (it was showing correctly when editing the file, and showing correctly on your comment, but not correctly on the diff (Files changed tab) 😄